### PR TITLE
Final Html structure and css

### PR DESCRIPTION
**Pull request by petudeveloper**
Make the skeleton of the webpage, add style to it using flexbox, float and grid.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,40 @@
...
+> In this project the assignment is to copy an article from the New York Times.
+
+## Built With
+
+- CSS
+- HTML
+
+## Live Demo
+
+[Live Demo Link](https://livedemo.com)
```
Comment (teekaytech) [inline README.md]: This link is broken.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,761 @@
...
+            <li class="footerItem">Accessibility</li>
+            <li class="footerItem">Work with us</li>
+            <li class="footerItem">Advertise</li>
+            <li class="footerItem">T Brand Studio</li>
+            <li class="footerItem">Your Ad Choices</li>
+            <li class="footerItem">Privacy Policy</li>
+            <li class="footerItem">Terms of Service</li>
+            <li class="footerItem">Site Map</li>
+            <li class="footerItem">Help</li>
+            <li class="footerItem">Subscriptions</li>
```
Comment (teekaytech) [inline index.html]: These items should be linked, using `#` as the value for their `href` attributes.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,761 @@
...
+                                <span>International</span>
+                            </li>
+                            <li class="normalItemSubNav topicList">
+                                <span>Canada</span>
+                            </li>
+                            <li class="normalItemSubNav topicList">
+                                <span>Español</span>
+                            </li>
+                            <li class="normalItemSubNav topicList">
+                                <span>中文网</span>
```
Comment (teekaytech) [inline index.html]: These items should be linked, using `#` as the value for their `href` attributes.

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello @petumazo  :wave:

__You've really done well so far. Here're the major highlights of this PR:__

- All linter checks are running/passing successfully :100: 
- Webpage looks good and similar to the original :ok_hand: 
- Descriptive PR title and summary :+1: 
- Nice implementation of `The Theory of Inflation` and `Editor's Picks` sections :1st_place_medal: 

__However, some changes are required before this project can be approved. Please check my comments below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

- [x] Update your Readme:
- Add project screenshot to the Readme, preferably after the project description.
-  The live demo link is broken. Since you've deployed with GitHub pages, just put the link in your Readme
- The `Issues page` and `MIT license` links are broken. Kindly fix them.
- [x] The NY Times logo should be linked (i.e. use `<a>` tag around the logo) (<a href="https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0" rel="nofollow noreferrer noopener" target="_blank">link to page</a>)
- [x] The title of articles/items in the `Suggested newsletters for you`, `Most Popular`, `More in Space and Astronomy` and the `Editors' Picks` should be linked. Any valid value e.g. "#" is okay for their href attribute.
- [x] Each of the items in the footer nav section should also be linked. Any valid value e.g. "#" is okay for their href attribute.
- [x] Always strive to keep your code DRY. 
- For instance, You're having the `display: flex;` rule in about 18 different blocks of your CSS rules. To keep your code DRY, you should consider having a block for this rule with its own class name (i.e. `.flex { display: flex; }`) You can then use this class name for all elements you wish to render as flex. This is possible, as you can have more than one class name for an element. You can then use another class name for other unique rules for each of the elements.

**[ OPTIONAL CHANGES ]**

- [ ] You should start practicing so that you can get used to using class selectors extensively for your CSS rules. You've used a lot of `id` selectors for your CSS rules. This will work, but it can mess up the cascading nature of your CSS rules. This is because the `id` selector has higher precedence than the class selector.
- [ ] You might want to reduce the font size of the `Most Popular` section. It seems pretty big.


* * *
**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review unless it is requested otherwise.**

**Always post your inquiries and questions on this PR thread as a comment. You can mention me (@teekaytech) so I can see your comments and respond.**

**REMINDER: As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 code reviews left for this project.**

_**Please make the required changes and submit another review request when you are done.**_

* * *

**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

**Comment by petudeveloper** [comment]
@teekaytech thank you so much for your comments and suggestions. They helped me a lot to understand how to clean my code.

**Review by smcommits — APPROVED** [review]
# Status: Approved :+1: 
## Hello, @petumazo! You have done a great job. The website shows an impressive amount of effort. Nice work on addressing all the issues pointed out by the previous TSE. This PR is now approved. You can go ahead and merge the branch. Congratulations and good luck with your next project.

**Comment by teekaytech** [comment]
> @teekaytech thank you so much for your comments and suggestions. They helped me a lot to understand how to clean my code.

Happy to help :blush:

**Pull request merged**